services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -131,6 +137,11 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
     }
   }, []);
 
+  useEffect(() => {
+    const reportType = initialValues?.permit_condition_category_code ? "PRR" : "CRR";
+    dispatch(change(FORM.VIEW_EDIT_REPORT, "report_type", reportType));
+  }, [!formValues?.report_type]);
```

Comment:
I used `!formValues?.report_type` as the `useEffect `condition here because at the moment if a user discards the form changes and we are keeping `report_type` radio buttons disabled, the user cannot make a selection. This should be adjusted in the work done for MDS-5781